Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build Stage
+FROM golang:1 as build
+
+ARG VERSION="1.14-develop"
```

Comment:
How does this work? Can we please instead read the version from the git repo?

Reply:
So right now the git repo isn't using any version, it has to be handed a version when it's compiled (see [here](https://github.com/GoogleCloudPlatform/cloudsql-proxy/blob/ea8931ea694c5e98eece94912a825e17e012ff70/cmd/cloud_sql_proxy/build.sh#L45))

This changes it so the argument can be passed in during the docker build command, and has a reasonable default so if compiled by hand it lets us know.